demisto_sdk/main.py

```diff
@@ -1,11 +1,241 @@
+                               validate_conf_json=kwargs['conf_json'], use_git=kwargs['use_git'])
+    return validator.run()
+
+
+@main.command(name="create",
+              short_help='Create content artifacts.')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-a', '--artifacts_path', help='The path of the directory in which you want to save the created content artifacts')
+@click.option(
+    '-p', '--preserve_bundles', is_flag=True, default=False,
+    help='Flag for if you\'d like to keep the bundles created in the process of making the content artifacts')
+def create(**kwargs):
+    content_creator = ContentCreator(**kwargs)
+    content_creator.run()
+
+
+@main.command(name="secrets",
+              short_help="Run Secrets validator to catch sensitive data before exposing your code to public repository."
+                         " Attach full path to whitelist to allow manual whitelists. Default file path to secrets is "
+                         "'./Tests/secrets_white_list.json' ")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-c', '--circle', type=click.Choice(["True", "False"]), default='False',
+    help='Is CircleCi or not')
+@click.option(
+    '-wl', '--whitelist', default='./Tests/secrets_white_list.json',
+    help='Full path to whitelist file, file name should be "secrets_white_list.json"')
+@pass_config
+def secrets(config, **kwargs):
+    sys.path.append(config.configuration.env_dir)
+    validator = SecretsValidator(configuration=config.configuration, is_circle=str2bool(kwargs['circle']),
+                                 white_list_path=kwargs['whitelist'])
+    return validator.run()
+
+
+@main.command(name="lint",
+              short_help="Run lintings (flake8, mypy, pylint, bandit) and pytest. pylint and pytest will run within the"
+                         "docker image of an integration/script. Meant to be used with integrations/scripts that use "
+                         "the folder (package) structure. Will lookup up what docker image to use and will setup the "
+                         "dev dependencies and file in the target folder. ")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-d", "--dir", help="Specify directory of integration/script", required=True)
+@click.option(
+    "--no-pylint", is_flag=True, help="Do NOT run pylint linter")
+@click.option(
+    "--no-mypy", is_flag=True, help="Do NOT run mypy static type checking")
+@click.option(
+    "--no-flake8", is_flag=True, help="Do NOT run flake8 linter")
+@click.option(
+    "--no-bandit", is_flag=True, help="Do NOT run bandit linter")
+@click.option(
+    "--no-test", is_flag=True, help="Do NOT test (skip pytest)")
+@click.option(
+    "-r", "--root", is_flag=True, help="Run pytest container with root user")
+@click.option(
+    "-k", "--keep-container", is_flag=True, help="Keep the test container")
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Verbose output")
```

Comment:
in the help we should mention that this flag is for debugging/troubleshooting purposes